tests/standardise/test_standardise.py

```diff
@@ -319,6 +319,7 @@ def test_standardise_flux():
     assert "co2_gpp-cardamom_europe" in proc_results
 
 
+@pytest.mark.skip(reason="Testing removal of this to find stuck test")
```

Comment:
We do want to be careful skipping tests but if we expect this to be fixed in v0.8.0 (new_object_store) this should included in the reason string. We'd also want to add an issue to make sure this gets added back in the next version - could even put this issue on the Special Release milestone.